modules/references/client/components/read-references/ReferencesReadPresentational.js

```diff
@@ -0,0 +1,93 @@
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Reference from './Reference';
+
+export default function ReferencesReadPresentational({
+  publicReferences,
+  nonpublicReferences,
+  user,
+}) {
+  const { t } = useTranslation('references');
+
+  const hasAnyPublicReferences =
+    publicReferences && publicReferences.length > 0;
+  const hasAnyNonpublicReferences =
+    nonpublicReferences && nonpublicReferences.length > 0;
+
+  // No references
+  if (!hasAnyNonpublicReferences && !hasAnyPublicReferences) {
+    return (
+      <div className="row content-empty">
+        <i className="icon-3x icon-users"></i>
+        <h4>{t('No references yet.')}</h4>
+        <a href={`/profile/${user.username}/references/new`}>
+          {t('Write one!')}
+        </a>
```

Comment:
"Write one!" link needs to be hidden if user is looking at their own profile. We should pass currently logged in user ID as a prop here.